src/version.rs

```diff
@@ -92,6 +92,12 @@ impl VersionInfo {
             .map(|p| (p.0.to_string(), p.1.to_string()))
             .collect()
     }
+
+    /// Determine if GDAL is compiled with [GEOS](https://libgeos.org/) support.
+    pub fn has_geos() -> bool {
+        version_info("BUILD_INFO").contains("GEOS_ENABLED=YES")
+    }
```

Comment:
@jdroenner @lnicola Should this be memoized somehow? I could imagine someone incorporating this into their logic (branching on capabilities) and this killing performance.

Reply:
I don't think it's necessary (I mean, what are you gonna do if these don't work?), but you can use https://docs.rs/once_cell/latest/once_cell/ if you want do it.

Reply:
Good point. Should probably keep it simple for now. I'll update the docs to reference it and re-request a review for final approval.

Reply:
PS: Plan on continuing the work of fleshing out the vector API in subsequent PRs.